docs/products/redis/concepts/ssl-connectivity.rst

```diff
@@ -0,0 +1,86 @@
+SSL connectivity
+================
+
+Client support for SSL-encrypted connections
+--------------------------------------------
+
+Default support
+~~~~~~~~~~~~~~~
+Aiven for Redis uses SSL encrypted connections by default. This is shown by the use of ``rediss://`` (with double s) prefix in the ``Service URI``. 
+
+.. Tip::
+    You can find the ``Service URI`` on `Aiven console <https://console.aiven.io/>`_.
+
+Since **Redis 6**, the ``redis-cli`` tool itself supports SSL connections; therefore, you can connect directly to your service using::
+
+    redis-cli -u rediss://username:password@host:port
+
+Alternatively, you can use the third-party `Redli tool <https://github.com/IBM-Cloud/redli>`_::
+
+    redli -u rediss://username:password@host:port
+
+
+Not every Redis client supports SSL-encrypted connections.
+In such cases, you would have to turn off SSL to use these clients, which is allowed but **not recommended**. You can use one of the following option to disable SSL.
```

Comment:
One of the two options doesn't really disable SSL but rather bypasses the SSL problem by removing the SSL bit locally.
```suggestion
In such cases, you would have to turn off or bypass the SSL to use these clients, which is allowed but **not recommended**. You can use one of the following option to achieve this.
```